style.css

```diff
@@ -109,12 +123,252 @@ h2 {
   color: solid #000;
   display: flex;
   list-style-type: none;
-  padding: 0;
+  padding: 0.05;
   justify-content: left;
   align-items: center;
   gap: 1.25rem;
 }
 
 .social-media a :hover {
-  fill: #3909c8;
+  color: #3909c8;
+}
+
+.work-section {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr;
+  grid-gap: 1.5rem;
+}
+
+.project-card {
+  padding: 0.75rem;
+  margin: 0.03 auto;
+  width: 100%;
+  max-width: 20.625rem;
+  border-radius: 0.05rem;
+}
+
+.project-title {
+  font-family: poppins, sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  letter-spacing: 0.03em;
+}
+
+.programming-languages {
+  color: solid #000;
+  display: flex;
+  list-style-type: none;
+  padding: 0.05;
+  justify-content: left;
+  align-items: center;
+  gap: 1.25rem;
+}
+
+.counter {
+  display: flex;
+  list-style-type: none;
+  padding: 0.05;
+  justify-content: left;
+  align-items: center;
+  gap: 1.25rem;
+}
+
+.canopy-text {
+  display: flex;
+  padding: 0.05;
+  justify-content: left;
+  align-items: center;
+  gap: 1.25rem;
+  font-family: poppins, sans-serif;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0.05em;
+  text-align: left;
+}
+
+.canopy {
+  font-family: poppins, sans-serif;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0.05em;
+  text-align: left;
+}
+
+.info-section {
+  padding-left: 1.5rem;
+  padding-right: 1.5rem;
+}
+
+.about-info {
+  max-width: 37.5rem;
+  margin: 1.5rem auto;
+}
+
+.info-container {
+  max-width: 37.5rem;
+  margin: 1.5rem auto;
+}
+
+.primary-text {
+  font-family: poppins, sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0.05em;
+  text-align: left;
+}
+
+.btn {
+  color: #396df2;
+  background: #efebeb;
+}
+
+.see-project-button {
+  color: #396df2;
+  font-family: poppins, sans-serif;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  border-radius: 25px;
+  border: 2px solid #396df2;
+  padding: 5px;
+  width: 200px;
+  height: 150px;
+}
+
+.list-frames {
+  list-style-type: none;
+  left: 12px;
+  top: 12px;
+  border-radius: 8px;
+}
+
+.language-framework {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 90rem;
+  margin: auto;
+  padding: 1rem;
+}
+
+.language-skills {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 90rem;
+  margin: auto;
+  padding: 1rem;
+}
+
+.framework {
+  font-family: poppins, sans-serif;
+  font-size: 20px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.05em;
+  text-align: left;
+}
+
+.skills {
+  font-family: poppins, sans-serif;
+  font-size: 20px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.05em;
+  text-align: left;
+}
```

Comment:
- [ ] Good work adding the differents skills! 🥇 Kindly in this section there are several differences, first make the down-arrow button have a background white and no borders, as in the template.

![image](https://user-images.githubusercontent.com/93630700/205455255-89f55de3-2f67-4d53-9605-1d4a74c1484b.png)

![image](https://user-images.githubusercontent.com/93630700/205455311-7ece8045-afb0-4ccf-9d2a-29180ca6bfc1.png)



- [ ] Kindly add the correct path to the icons so they can be displayed on the web 💯 
- [ ] Kindly give the text the corresponding font family and size to make it looks similar to the FIGMA template
- [ ] The arrows that point to the right are missing, kindly fix the to be showed on the web 💯